.github/workflows/test.yaml

```diff
@@ -15,7 +15,11 @@ jobs:
     - uses: actions/checkout@v3
 
     - name: Install dependencies
-      run: pip install -r requirements.txt
+      run: |
+        python --version
+        pip install --upgrade pip
+        pip --version
```

Comment:
I stand by leaving this in: if the requirements install correctly, no one will notice this, and if they install incorrectly, this will be useful information to have.